### DISCUSSION TITLE
Snap out of date

### DISCUSSION THREAD

**Discussion by theinfguy** [seerr-team/seerr#819] [General]
Hi there, just letting you know your snap build is 9 months out of date :-(

https://snapcraft.io/install/jellyseerr/ubuntu

**Comment by fallenbagel** [comment]
We don't update snap anymore.

Snap builds are broken so until we can fix that, snap is deprecated for now

**Comment by theinfguy** [comment]
Thanks, unless I’m missing it, that would have been nice to know.On 16 Jun 2024, at 09:13, Fallenbagel ***@***.***> wrote:﻿
We don't update snap anymore.
Snap builds are broken so until we can fix that, snap is deprecated for now
